browser-test/tests/repair.spec.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.skip('bundled-ses lockdown is consistent across browsers', async ({
+  page,
+  browser,
+  browserName,
+}) => {
+  page.on('pageerror', error => {
+    console.error(`________________
+> Error in page: ${error.message}
+${error.stack}`);
+  });
+  await page.goto(`http://127.0.0.1:3000/`);
+  console.log(browserName, browser.version());
+  // Expect a title "to contain" a substring.
+  // await expect(page).toHaveTitle(/Pass/);
+  const result = await page.evaluate(() => {
+    Object.defineProperty(Object.prototype, 'toString', {
+      value: function () {},
+    });
+    try {
+      lockdown();
+    } catch (e) {
+      return e.message;
```

Comment:
Fails with ` "Cannot delete property 'arguments' of function () {}"` only in chromium

Reply:
attn @erights. My understanding of the above code is that `page.evaluate` calls `code.toString()` and evaluates that in the page. I assume that means this is running in sloppy mode so the overridden `toString` has an `arguments` object. It’s unclear to me why only Chromium would reäct to this code in this way. Maybe a `"use strict"` would make all engines perform uniformly. Is it possible that this is a variation in behavior for the Chromium harness?

Reply:
@erights I see your reaction from an hour ago. Thanks.

Reply:
Adding `"use strict"` in front of the function does make it consistent again.